tests/Bridge/Doctrine/Orm/Extension/PaginationExtensionTest.php

```diff
@@ -887,6 +888,35 @@ public function testGetResult()
         $this->assertInstanceOf(PaginatorInterface::class, $result);
     }
 
+    public function testGetResultWithoutDistinct()
+    {
+        $configuration = new Configuration();
+
+        $entityManagerProphecy = $this->prophesize(EntityManagerInterface::class);
+        $entityManagerProphecy->getConfiguration()->willReturn($configuration)->shouldBeCalled();
```

Comment:
Actually, `shouldBeCalled` should not be called, unless we actually need to make such an assertion.

Reply:
We need a PR to remove useless assertions. Lol...

Reply:
These are part of the test IMO there where missing and I added them for consistency with the legacy fn.